spec/controllers/builds_controller_spec.rb

```diff
@@ -123,6 +123,19 @@
     end
   end
 
+  describe "#show" do
+    it "should return a valid JSON" do
+      git_sha = 'a7c751f2c408e585ea8d95a58d474d08d7fd5c2f'
```

Comment:
since the `git_sha` is not referenced in the test you should just let FactoryGirl generate one for you